### PR TITLE
#115 Throw errors when model name and unit dynamic model are the same…

### PR DESCRIPTION
**Pull request by rosiereflo**
… or if dynamic model id and modelica class name are the same

### CODE REVIEW

File: dynawo/sources/Solvers/SolverIDA/test/jobs/solverTestAlpha.dyd [hunk 1]
```diff
@@ -13,6 +13,6 @@
 -->
 <dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
   <dyn:modelicaModel id="solverTestAlpha">
-    <dyn:unitDynamicModel id="solverTestAlpha" name="DYNSolverTestAlpha" moFile="DYNSolverTest.mo"/>
+    <dyn:unitDynamicModel id="solverTestAlphaModel" name="DYNSolverTestAlpha" moFile="DYNSolverTest.mo"/>
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Solvers/SolverIDA/test/jobs/solverTestAlpha.dyd]: We started to use the following coding style for this kind of things:

- modelicaModel id starts with a capital letter
- unitDynamicModel id starts with a small letter (that way, no need to add the suffix Model)

Same comment for the other tests

File: dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic [hunk 2]
```diff
@@ -207,6 +207,8 @@ NumericalErrorFunction      =             A numerical error occurs during the ca
 //---CSTR--------------------
 //---DYD---------------------
 ModelIDNotUnique            =             model id (%1%) is not unique
+UnitModelIDSameAsModelName  =             model %1% has the same name as the parent modelica model
```
Comment (MarianneSaugierRTE) [inline dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic]: I would rather use a vocabulary closer to the xml terminology in order to clearly understand what is the parent modelica model, the modelica class, the model name etc. I suggest the following error messages:
- unitDynamicModel %1%  id should be different from modelicaModel id
- unitDynamicModel %1% id should be different from its name

What do you think?

Comment (rosiereflo) [reply dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic]: Sounds good!

File: dynawo/sources/Solvers/SolverIDA/test/jobs/solverTestAlpha.dyd (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/Solvers/SolverIDA/test/jobs/solverTestAlpha.dyd]: Solved

File: dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic (hunk 2 above)
Comment (rosiereflo) [reply dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic]: Solved

**Pull request merged**